### ISSUE TITLE
Error retrieving icon function

### ISSUE THREAD

**Issue by shamindras** [rougier/svg-lib#20]
@rougier - thanks for creating such a cool package.

I tried to install this using [straight.el](https://github.com/radian-software/straight.el) as follows locally:

```lisp
(use-package svg-lib
      :straight '(svg-lib
		  :host github
		  :repo "rougier/svg-lib"))
```

It seems to run without error. However when I try to use this with `corfu` which relies on the svg-icons, I keep getting the warning:

```
Warning (emacs): Error retrieving icon function, falling back on short-text
(svg-lib.el requires Emacs to be compiled with svg support.
) Disable showing Disable logging
```

Unfortunately the `svg-lib` icons don't show as a result. I've attached a screenshot of the situation below:

<img width="1417" alt="Screen Shot 2022-06-13 at 12 38 54 PM" src="https://user-images.githubusercontent.com/7627188/173403220-3de16082-776f-4140-b8ec-b385cced485b.png">

As you can see the pop up dialog from `corfu` does not have the icons but just the fallback text i.e. `kw` and `f` etc.

I have the following questions:
1. Is there something I'm doing wrong with installing the `svg-lib` package? 
2. Are there any other packages I need to install on my machine before installing on emacs?

I'm running this on macOS (Monterey v12.2.1) and using Emacs v28.1 on a macbook with 64GB ram.

I'm relatively new to emacs, so apologies if this is covered elsewhere. I just wasn't able to solve it.

**Comment by rougier** [comment]
Yes, it might be worth to have a paragraph about installation stating that Emaccs svg support is required (with maybe a link to emacs-mac or emacs-plus for OSX users). Since the package is on ELPA, you'll need to sign the copyright assignment (see https://www.gnu.org/software/emacs/manual/html_node/emacs/Copyright-Assignment.html)
